posthog/api/insight_serializers.py

```diff
@@ -51,7 +51,26 @@ def __init__(self, request=None, *args, **kwargs):
     actions = FilterActionSerializer(
         required=False, many=True, help_text="Actions to filter on. One of `events` or `actions` is required."
     )
-    properties = PropertySerializer(many=True, required=False)
+    properties = PropertyGroupField(
+        required=False,
+        help_text="""
+        Properties to filter on. A JSON object that looks like:
+        {
+            "type": "AND",
+            "values": [ Properties ]
+        }
+
+        where a property looks like:
+        {
+            "key": "email",
+            "value": "x@y.com",
+            "operator": "exact",
+            "type": "event"
+        }.
+
+        Can be nested arbitrarily.
+        """,
```

Comment:
You can use markdown here! Also you don't want to indent this stuff
```suggestion
        help_text="""Properties to filter on. A JSON object that looks like:
```json
{
    "type": "AND",
    "values": [ Properties ]
}
```

where a property looks like:
```json
{
    "key": "email",
    "value": "x@y.com",
    "operator": "exact",
    "type": "event"
}
```

Can be nested arbitrarily.""",
```